File: app/src/at-nhatnguyen/java/asiantech/internship/summer/javacore/Bai18.java
package asiantech.internship.summer.javacore;

import java.util.Scanner;
import java.util.StringTokenizer;
public class Bai18 {
    public static void main(String[] args)
    {
        Scanner scan = new Scanner(System.in);
        System.out.println("Nhap vao 1 xau: ");
        String str = scan.nextLine();
        StringTokenizer strToken = new StringTokenizer(str, " ");
        System.out.println("So tu trong xau la: "+strToken.countTokens());
    }
}

File: app/src/at-trinhnguyen/java/asiantech/internship/winter/javacore/congtrinh/Bai18.kt
package com.congtrinh

import java.util.*

/*
Bài 18. Nhập một xâu ký tự. Đếm số từ của xâu ký tự đó. Thí dụ " Trường học " có 2 từ.
 */
object Bai18 {
    @JvmStatic
    fun main(args:Array<String>){
        val scanner=Scanner(System.`in`)
        print("Nhap vao mot chuoi: ")
        val str=scanner.nextLine()
        println("So tu co trong cau la: ${StringTokenizer(str," ").countTokens()}")
    }
}